app/managers.py

```diff
@@ -0,0 +1,51 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.db3")
+        self.table_name = "actors"
+
+    def all(self) -> list:
+        actor = [
+            Actor(*actor)
+            for actor
+            in self._connection.execute(f"SELECT * FROM {self.table_name}")
+        ]
+        return actor
+
+    def create(
+            self,
+            first_name: str,
+            last_name: str
+    ) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self.table_name} (first_name, last_name) "
+            "VALUES (?, ?)", (first_name, last_name)
+        )
+        self._connection.commit()
+
+    def update(
+            self,
+            id_for_update: int,
+            new_first_name: str,
+            new_last_name: str
+    ) -> None:
+        self._connection.execute(
+            f"UPDATE {self.table_name} SET first_name = ?, last_name = ?"
+            "WHERE id = ?",
+            (new_first_name, new_last_name, id_for_update)
+        )
+        self._connection.commit()
+
+    def delete(
+            self,
+            id_to_delete: int
+    ) -> None:
+        self._connection.execute(
+            f"DELETE FROM {self.table_name} WHERE id = ?",
```

Comment:
It's better to follow SQL style guidelines even while writing raw SQL in Python. Therefore I'd suggest to start WHERE clause from new line.